Add element for SplitLayout

alvarezguille opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-components-testbench/32)
<!-- Reviewable:end -->

alvarezguille committed:
Add element for split-layout

oluwasayo commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/vaadin/vaadin-components-testbench/32)*
<!-- Sent from Reviewable.io -->

alvarezguille committed:
Add simple test for splitter

Original test used drag and drop which caused some issues when running on remote Firefox

oluwasayo commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/vaadin/vaadin-components-testbench/32)*
<!-- Sent from Reviewable.io -->
